website/pages/docs/secrets/databases/mssql.mdx

```diff
@@ -46,6 +46,7 @@ more information about setting up the database secrets engine.
         username="vaultuser" \
         password="yourStrong(!)Password"
     ```
+    > Note: The example above demonstrates a connection with SQL server user named `vaultuser`, although the user `vaultuser` might be Windows Authentication user part of Active Directory domain, for example: `DOMAIN\vaultuser`.
```

Comment:
```suggestion
    ~> Note: The example above demonstrates a connection with SQL server user named `vaultuser`, although the user `vaultuser` might be Windows Authentication user part of Active Directory domain, for example: `DOMAIN\vaultuser`.
```

Reply:
You could also do `->` which makes for a blue background. You can see an example of this at the top of the Transform secrets docs: https://deploy-preview-9743--vault-www.netlify.app/docs/secrets/transform